packages/search/src/utils.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * A `React.useEffect` that will not run on the first render.
+ *
+ * Source:
+ * https://github.com/reakit/reakit/blob/HEAD/packages/reakit-utils/src/useUpdateEffect.ts
+ *
```

Comment:
Not a blocker, but I'd also add a short paragraph to exemplify a real-world case where it'd make sense to use this hook.